projects/plugins/jetpack/extensions/blocks/coauthor-paragraph/index.js

```diff
@@ -0,0 +1,82 @@
+import { useBlockProps } from '@wordpress/block-editor';
+import { createBlock } from '@wordpress/blocks';
+import { Fragment } from '@wordpress/element';
+import { __ } from '@wordpress/i18n';
+import { getIconColor } from '../../shared/block-icons';
+import attributes from './attributes';
+import edit from './edit';
+
+/**
+ * Style dependencies
+ */
+import './editor.scss';
+
+export const name = 'coauthor-paragraph';
+export const title = __( 'Coauthor Paragraph (Experimental)', 'jetpack' );
+export const settings = {
+	apiVersion: 2,
+	title,
+	description: (
+		<Fragment>
+			<p>
+				{ __(
+					'Automatically generate new paragraphs using your existing content, powered by AI magic. We are experimenting with this feature and can tweak or remove it at any point.',
+					'jetpack'
+				) }
+			</p>
+		</Fragment>
+	),
+	icon: {
+		src: 'welcome-write-blog',
+		foreground: getIconColor(),
+	},
+	category: 'text',
+	keywords: [ 'AI', 'GPT' ],
```

Comment:
Those will need to be translated as well. I wonder if folks will be searching for "AI" or "GPT" in their editor, rather than "help" or "assistant".

Reply:
Yes, I absolutely believe people will be searching for those terms specifically regardless of the language they will be using.